src/components/header.js

```diff
@@ -1,65 +1,145 @@
+      ></div>
+      <div className="header-container">
+        <h1 className="header-container__title">
+          <Link to="/" className="header-container__navLink">
+            <div
+              className={`header-container__logo ${
+                mobileExpanded ? "header-container__logo--open" : ""
+              }`}
+            ></div>
+          </Link>
+        </h1>
+        <nav className="nav" ref={mobileNavRef}>
+          <ul className="nav-list">
+            <li className="nav-list__item">
+              <Link
+                className={active === "about" ? activeLinkStyles : linkStyles}
+                to="/about"
+              >
+                About
+              </Link>
+            </li>
+            <li className="nav-list__item">
+              <Link
+                className={
+                  active === "programs" ? activeLinkStyles : linkStyles
+                }
+                to="/programs"
+              >
+                Programs
+              </Link>
+            </li>
+            <li className="nav-list__item">
+              <Link
+                className={
+                  active === "resources" ? activeLinkStyles : linkStyles
+                }
+                to="/resources"
+              >
+                Resources
+              </Link>
+            </li>
+            <li className="nav-list__item">
+              <Link
+                className={active === "news" ? activeLinkStyles : linkStyles}
+                to="/news"
+              >
+                News
+              </Link>
+            </li>
+            <li className="nav-list__item">
+              <Link
+                className={active === "contact" ? activeLinkStyles : linkStyles}
+                to="/contact"
+              >
+                Contact
+              </Link>
+            </li>
+          </ul>
+        </nav>
+        {/* <nav className="nav-mobile" ref={mobileNavRef}>
+          <ul className="nav-list-mobile">
+            <li className="nav-list__item nav-list-mobile__item">
+              <Link className={linkStyles} to="/about">
+                About
+              </Link>
+            </li>
+            <li className="nav-list__item nav-list-mobile__item">
+              <Link className={linkStyles} to="/programs">
+                Programs
+              </Link>
+            </li>
+            <li className="nav-list__item nav-list-mobile__item">
+              <Link className={linkStyles} to="/resources">
+                Resources
+              </Link>
+            </li>
+            <li className="nav-list__item nav-list-mobile__item">
+              <Link className={linkStyles} to="/news">
+                News
+              </Link>
+            </li>
+            <li className="nav-list__item nav-list-mobile__item">
+              <Link className={linkStyles} to="/contact">
+                Contact
+              </Link>
+            </li>
+          </ul>
+        </nav> */}
```

Comment:
Don't forget to remove this.

Reply:
Opened an issue for this! Since targeting classNames breaks the animations.